docs/conf.py

```diff
@@ -239,6 +240,8 @@
     "term": "tooltip",
 }
 
+# Panels options
+
```

Comment:
Do we need this?

Reply:
No, this was left over from my panels experiment.